### DISCUSSION TITLE
Colorable higher tiered backpacks [feature request]

### DISCUSSION THREAD

**Discussion by Charmanth** [P3pp3rF1y/SophisticatedBackpacks#244] [Ideas]
Is it possible to make it so iron+ backpacks in this mod are colorable?  They look like the default leather which makes me a bit sad.

**Comment by P3pp3rF1y** [comment]
All of the backpacks can be dyed and dyed backpacks can be upgraded. They only are not added to JEI / vanilla creative as that would mean 68 additional items in there for the mod which are not necessary. I may just take one more look at figuring out if dye recipes can be shown in JEI in some way.

**Comment by Charmanth** [comment]
I meant to come back and comment but forgot.  We (meaning my friends and I) literally just figured this out right after I posted and today (both parts of figuring out were, ah, lightbulb moments for us).  Anyways, thanks for thinking ahead that this is something people would want, and yeah if you can get it shown in JEI somehow that would be great!  Maybe a tagline on the higher tiers (can be colored)?
